community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -544,7 +551,8 @@ object projects:
     project = "cats",
     sbtTestCommand = "set scalaJSStage in Global := FastOptStage;buildJVM;validateAllJS",
     sbtPublishCommand = "catsJVM/publishLocal;catsJS/publishLocal",
-    dependencies = List(discipline, disciplineMunit, scalacheck, simulacrumScalafixAnnotations)
+    dependencies = List(discipline, disciplineMunit, scalacheck, simulacrumScalafixAnnotations),
+    scalacOptions = Nil  // disable -Ycheck-init, due to -Xfatal-warning
```

Comment:
@anatoliykmetyuk Could you have a look at the changes in this file?

Reply:
This file LGTM!

Reply:
Thanks @anatoliykmetyuk !